homeassistant/components/iotawatt/coordinator.py

```diff
@@ -30,6 +34,13 @@ def __init__(self, hass: HomeAssistant, entry: ConfigEntry) -> None:
             logger=_LOGGER,
             name=entry.title,
             update_interval=timedelta(seconds=30),
+            request_refresh_debouncer=Debouncer(
+                hass,
+                _LOGGER,
+                cooldown=REQUEST_REFRESH_DEFAULT_COOLDOWN,
+                immediate=True,
+                function=self.async_refresh,
```

Comment:
```suggestion
```

This will get filled automatically by the `DataUpdateCoordinator`

https://github.com/home-assistant/core/blob/dev/homeassistant/helpers/update_coordinator.py#L77